Features5/tests/features/deleteContact.feature

```diff
@@ -0,0 +1,43 @@
+Feature: Delete Contact
+     As a user
+     I want to delete contact
+     So that I can free up the space
+    
+    Scenario Outline: User wants to delete particular contact
+        Given user has navigated to contact lists in the homepage containing following contacts:
+        |name    |email    |phone     |contactType   |
+        |<name>  |<email>  |<phone>   |<contactType> |
```

Comment:
```suggestion
        Given user has navigated to homepage
        And user has added following contacts:
        |name    |email    |phone     |contactType   |
        |<name>  |<email>  |<phone>   |<contactType> |
```

break this down like above and also to others feature file where you have used liked this